toolchest_client/tools/tool_args.py

```diff
@@ -206,6 +206,29 @@
             "--chemistry": 1,
         },
     },
+    "centrifuge": {
+        "whitelist": {
+            # This allows all args that are not in the blacklist if this is the only whitelisted arg
+            "*": 0
+        },
+        "blacklist": [
+            "-S",
+            "-p",
+            "--threads",
+            "--un",
+            "--al",
+            "--un-conc",
+            "--al-conc",
+            "--un-gz",
+            "--un-bz2",
+            "--al-gz",
+            "--al-bz2",
+            "--un-conc-gz",
+            "--un-conc-bz2",
+            "--al-conc-gz",
+            "--al-conc-bz2",
+        ]
```

Comment:
Does this config actually reject non-blacklisted args that aren't the first item in `tool_args`?

Reply:
I'm not sure what you mean by this, can you give me an example config? (I don't believe so if I'm understanding correctly, but I just want to make sure that I am.)